Resources/Prototypes/_Mono/Entities/SpaceArtillery/SpaceArtillery/Energy/launcher.yml

```
@@ -547,3 +548,348 @@
         density: 1500
   - type: ShipGunType
     shipType: Energy
+
+# PRE- FRACTURE
+# Tachyon Lance "Reaper"
+
+- type: entity
+  id: WeaponTurretReaper
+  name: Tachyon Lance "Reaper"
```

```suggestion
  name: [type designation] tachyon lance "Reaper"
```
capitalization, naming scheme.